services/hanke-service/src/main/resources/email/template/attributes.mjml

```diff
@@ -0,0 +1,5 @@
+<mj-attributes>
```

Comment:
The mjml files could be moved elsewhere. Under resources, they will be a part of the runtime package, which they clearly don't need to be.

Reply:
Do you have any suggestions on where they should be located?

Reply:
How about an emailtemplate directory under scripts?

Reply:
They're not scripts, so that doesn't feel correct. Emails directory in either project root or `services/hanke-service/emails`?

Reply:
Placed them in project root email/.

Reply:
👍 Dividing the common parts to a separate directory was a good addition.